modules/ROOT/pages/scale-cluster.adoc

```diff
@@ -0,0 +1,128 @@
+<   replicaCount: 1
+---
+>   replicaCount: 4
+>   volumes:
+>     journal:
+>       size: 2Gi
+>     ledgers:
+>       size: 10Gi
+>     ranges:
+>       size: 1Gi
+52,54c59,61
+<     defaultEnsembleSize: 1
+<     defaultAckQuorum: 1
+<     defaultWriteQuorum: 1
+---
+>     defaultEnsembleSize: 3
+>     defaultAckQuorum: 2
+>     defaultWriteQuorum: 3
+----
+
+To install from the above YAML:
+
+----
+$ helm install pulsar -f ~/dev-values_large.yaml --wait datastax-pulsar/pulsar
+----
+
+== Generate data
+
+. Generate data in our namespaces:
+
+----
+$ bin/pulsar-admin namespaces set-retention public/default -t -1 -s 100M
+$ bin/pulsar-client produce test1 -n 500 -m "sdasdsdsadsadsad"
+$ bin/pulsar-client produce test2 -n 500 -m "sdasdsdsadsadsad"
+$ bin/pulsar-client produce test3 -n 500 -m "sdasdsdsadsadsad"
+----
+[start=2]
+. Ensure all bookies have some ledgers assigned to them:
+
+----
+$ ./bin/bookkeeper shell listledgers -meta | grep pulsar-bookkeeper-3
+----
+
+== Destroy Persistent Volume Claim (PVC)
+
+To downscale a bookie and destroy its persistent volume claim (PVC):
+
+----
+ bookkeeper:
+    replicaCount: 3
+
+$ helm upgrade pulsar -f ~/dev-values_large.yaml --wait datastax-pulsar/pulsar
+----
+
+== Delete PVC
+
+To delete the PVC:
+----
+$ kubectl delete pvc pulsar-bookkeeper-ledgers-pulsar-bookkeeper-3
+----
+
+== Scale Back Cluster
+To scale back the cluster:
+----
+ bookkeeper:
+    replicaCount: 4
+
+$ helm upgrade pulsar -f ~/dev-values_large.yaml --wait datastax-pulsar/pulsar
+----
+
+Now, bookkeeper-3 will fail to start with
+`org.apache.bookkeeper.bookie.BookieException$InvalidCookieException`.
+
+== Recovery from lost PVC
+
+. Track under-replicated ledgers from any live bookie:
+----
+$ ./bin/bookkeeper shell listunderreplicated
+----
+[start=2]
+. Double-check bookie id of the failing bookie:
+----
+$ ./bin/bookkeeper shell listbookies -a
+----
+
+This should return `Cannot resolve pulsar-bookkeeper-3.pulsar-bookkeeper.default.svc.cluster.local:3181, bookie is unknown`.
+[start=3]
+. Decommission the bookie:
+
+----
+$ ./bin/bookkeeper shell decommissionbookie -bookieid pulsar-bookkeeper-3.pulsar-bookkeeper.default.svc.cluster.local:3181
+----
+[start=4]
+. Cookie of the decommissioned bookie: `pulsar-bookkeeper-3.pulsar-bookkeeper.default.svc.cluster.local:3181` is deleted successfully.
+. Restart failing bookie with Ctrl-K in k9s, or wait for k8s to restart it again.
+. The bookie should come up and rejoin the cluster (data is lost, as expected with deleted PVC).
+
+== Better scale down procedure
+
+1. Ensure there are no under replicated ledgers
+2. Downscale the cluster. New under replicated ledgers will appear.
+3. Decommission the removed bookie (bookie has to be down). This will take some time because it triggers the audit and autorecovery, and blocks until autorecovery completes. 
+4. There is an option to scale back if something fails
+5. Delete the PVCs of decommissioned bookie (both ledger and journal).
```

Comment:
```suggestion
. Ensure there are no under-replicated ledgers.
. Downscale the cluster.
+ 
New under replicated ledgers will appear.
. Decommission the removed bookie.
+
[NOTE]
====
The decommissioning will take some time because it triggers the audit and auto-recovery and blocks until auto-recovery completes. 
====
+
[NOTE]
====
There is an option to scale back if something fails.
====

. Delete the ledger and journal PVCs of decommissioned bookie.
```